src/main/webapp/dashboard.js

```diff
@@ -202,8 +202,12 @@ function getProject(projectID) {
  * Call functions to populate page sections with data.
  */
 async function setUpPage() {
-  const user = await getUser();
-  // TODO: redirect to sign up/in page if the user is not found.
+  const user = await getUser()
+      .catch((errorResponse) => {
+        console.error(
+            `Error ${errorResponse.statusCode}: ${errorResponse.message}`);
+        alert(errorResponse.userMessage);
```

Comment:
do we want to display an alert? Technically we don't expect this code to be hit, since they should be redirected first right?

Reply:
That's a good point! The redirect is a bit slow sometimes and I thought that maybe displaying a message to explain what is going on might be useful for the user. It is kind of pointless though so I'll get rid of it.

Reply:
We actually got a couple bad reviews from this feature haha. Good catch!